### DISCUSSION TITLE
How to remove word index in custom dictionaries?

### DISCUSSION THREAD

**Discussion by BadPackage** [tadashi-aikawa/obsidian-various-complements-plugin#312] [Q&A] [answered]
Hello,

I've tried adding these word lists to the custom dictionary, but they contain some indexes (e.g., `you 28787591`), or whatever the correct term for this might be. The problem is that these numbers are included in the suggestions. How can I remove these numbers effectively?

I attempted to use the [Word regex pattern](https://tadashi-aikawa.github.io/docs-obsidian-various-complements-plugin/4.%20Options/4.6.%20Custom%20dictionary%20complement/%E2%9A%99%EF%B8%8FWord%20regex%20pattern/), but I'm not sure if this is the right approach or if I am using it incorrectly. Any guidance would be appreciated!

I could, of course, use the word lists locally and manually clean them with a script, but I want to know if there's a more efficient or automated way to handle this within the plugin.

Here are the word lists I tried to use:
- [English Word List](https://github.com/hermitdave/FrequencyWords/raw/master/content/2018/en/en_50k.txt)
- [German Word List](https://github.com/hermitdave/FrequencyWords/raw/master/content/2018/de/de_50k.txt)

Thank you for your help!

**Comment by tadashi-aikawa** [comment, marked as answer]
Hi, @BadPackage :)

> I attempted to use the Word regex pattern, but I'm not sure if this is the right approach or if I am using it incorrectly. Any guidance would be appreciated!

The "Word regex pattern" is an option that controls the addition of entries to the custom dictionary on a line-by-line basis. It is not designed to extract patterns from within a line.

> I could, of course, use the word lists locally and manually clean them with a script, but I want to know if there's a more efficient or automated way to handle this within the plugin.

Downloading the file locally and processing it manually is the best approach, and there seems to be no other alternative. While adding whitespace as a delimiter is one option, whitespace often exists as part of the entries themselves, so I'm uncertain if it's the right choice.
